dwave/preprocessing/composites/clip.py

```diff
@@ -43,13 +43,6 @@ class ClipComposite(ComposedSampler):
        >>> sampler = ClipComposite(ExactSolver())
        >>> response = sampler.sample_ising(h, J, lower_bound=-2.0, upper_bound=2.0)
 
-    .. note::
-        Context manager usage pattern is recommended for samplers and composites
-        that allocate scope-bound resources, like :class:`~dwave.system.DWaveSampler`.
```

Comment:
"that allocate scope-bound resources, like :class:`~dwave.system.DWaveSampler`."

I was just about to give you some [reading material](https://www.merriam-webster.com/grammar/read-this-if-you-hate-the-word-like) I'm sure you'll enjoy